CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and Yorkie adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
+## [0.4.2] - 2023-06-19
+
+### Added
+* Add TLS Option & Insecure Flag in Admin CLI by @krapie in https://github.com/yorkie-team/yorkie/pull/548
+* Implement Tree.Style for setting attributes to nodes by @hackerwins in https://github.com/yorkie-team/yorkie/pull/549
+
+### Changed
+* Add missing Secrets configuration by @jongwooo in https://github.com/yorkie-team/yorkie/pull/553 
+* Abstract the workflow to base-docker-publish.yml by @jongwooo in https://github.com/yorkie-team/yorkie/pull/552
```

Comment:
```suggestion
* Abstract the workflow to base-docker-publish.yml by @jongwooo in https://github.com/yorkie-team/yorkie/pull/552
```

Reply:
Should I remove line 17 on the release as well?

Reply:
Yes, please. #553 fixes the bug caused by #552.